server/services/ticket.js

```diff
@@ -29,22 +30,24 @@ function createTicket(ticketDetails) {
   return new Promise((resolve, reject) => {
     Joi.validate(ticketDetails, ticketSchema, (err, value) => {
       if (err) {
-        reject(err);
-      } else {
-        resolve(Ticket.create(value));
+        return reject(err);
       }
+      resolve(Ticket.create(value));
     });
+  })
+  .then((ticket) => {
+    // alertService.createAlert(ticket);
```

Comment:
Commented out right now until I add in `.catch()` logic to stop tests from failing.